### ISSUE TITLE
RAP and HRRR forecasts are being extended for certain forecast cycles

### ISSUE THREAD

**Issue by nwagenbrenner** [firelab/windninja#365]
https://www.weather.gov/media/notification/scn20-46rapv5_hrrr_v4.pdf

Scheduled for May 26. Do we need to update the GUI or nomads settings?

**Comment by ksshannon** [comment]
I don't think we can have a dynamic value for time here.  We may just add another 'model' to handle the extended versions?  Something like `HRRR-EXT-XXXX` and `RAP-EXT-XXXX`?  Thoughts?

**Comment by nwagenbrenner** [comment]
Yeah, I think that's probably the best way to do it.

**Comment by ksshannon** [comment]
Started on this in scn20-46 branch.  I'll add all the ext models, and then wait for it to go in on NOMADS.  When it is merged, I'll close this.

**Comment by ksshannon** [comment]
This has been delayed.  I'll check in here when the models are updated.

**Comment by nwagenbrenner** [comment]
IMETs have requested the extended HRRR recently. We should look into adding this again.

**Comment by masonwillman** [comment]
Commit 1a2fe4c38159d35f3b7f00dc939cd9839cfa605e in branch [scn20-46](https://github.com/firelab/windninja/tree/scn20-46) adds RAP Conus extended. HRRR Conus Extended was added by Kyle Shannon in commit c67d391d4181e7a8edc60465fd6b1ba55506b7ea. Kyle also added testing in this commit ba8d6419ec3697c667b805214d5f11b6b01bab21. @nwagenbrenner should I continue to add testing for RAP Conus Extended, or is this old testing code that is no longer used?
